backend/typescript/rest/camperRoutes.ts

```diff
@@ -0,0 +1,39 @@
+import { Router } from "express";
+import { updateCamperDtoValidator } from "../middlewares/validators/camperValidators";
+import CamperService from "../services/implementations/camperService";
+import ICamperService from "../services/interfaces/camperService";
+import { getErrorMessage } from "../utilities/errorUtils";
+
+const camperRouter: Router = Router();
+
+const camperService: ICamperService = new CamperService();
+
+/* Update the camper with the specified camperId */
+camperRouter.put("/:camperId", updateCamperDtoValidator, async (req, res) => {
+  try {
+    const updatedCamper = await camperService.updateCamperById(
+      req.params.camperId,
+      {
```

Comment:
can we actually pass in camp to updateCamperById? admins will be able to move campers between session of the same camp